test/unit/host_test.rb

```diff
@@ -710,8 +710,6 @@ def teardown
       hostgroup = FactoryGirl.create(:hostgroup, :with_puppet_orchestration)
       h = FactoryGirl.create(:host, :managed, :with_environment, :hostgroup => hostgroup)
       Setting[:manage_puppetca] = true
-      assert h.puppet_proxy.present?
-      assert h.puppetca?
```

Comment:
The removal of these lines is perhaps indicating that the way the model worked has changed, as they previously would have been inherited.